### PR TITLE
reset some changes from commit cd289ee

### PR DESCRIPTION
**Pull request by LavrovArtem**
\cc @inikulin @churkin 
@VasilyStrelyaev please check comment

### CODE REVIEW

**Comment by inikulin** [comment]
lgtm

File: src/processing/script/instrumented.js [hunk 1]
```diff
@@ -71,15 +71,18 @@ const INSTRUMENTED_PROPERTY_RE = new RegExp(`^(${PROPERTIES.join('|')})$`);
 //    __set$(z, A, x.protect()); // x.protect - returns the removed method
 // The __set$ function calls the test method of the regular expression. (GH-331)
 var reTest = RegExp.prototype.test;
+// NOTE: The Function.prototype.call method also can be removed.
+// But at a time can be removed only one of the methods.
```
Comment (VasilyStrelyaev) [inline src/processing/script/instrumented.js]:
```
// NOTE: The Function.prototype.call method can also be removed.
// But only one of the methods can be removed at a time.
```

**Comment by churkin** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Pull request merged**